packages/amplify-cli/src/extensions/amplify-helpers/envResourceParams.ts

```diff
@@ -75,7 +75,12 @@ export function saveEnvResourceParameters(context: $TSContext, category: string,
     stateManager.setTeamProviderInfo(undefined, teamProviderInfo);
     // write hostedUIProviderCreds to deploymentSecrets
     const deploymentSecrets = stateManager.getDeploymentSecrets();
-    const rootStackId = getRootStackId();
+    let rootStackId;
+    try {
```

Comment:
Based on usage of `getRootStackId` we could do better, other than throwing, then swallowing. Other callers are doing something similar which could be enhanced inside that function.